kustomization.yaml

```diff
@@ -0,0 +1,80 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+images:
```

Comment:
Given this is a `base` layer, it's a bit surprising to see image overrides here. Why aren't these encoded in the manifests directly in the first place?

Reply:
We do not have build pipeline to get official image repository and image tags. The current ones are placeholders and will be updated once we have fully integrated with CI/CD.